eng/common/pipelines/templates/steps/prepare-pipelines.yml

```diff
@@ -13,125 +13,128 @@ parameters:
   - name: TestsConventionOptions
     type: string
     default: ''
+jobs:
+- job: PreparePipelines
+  pool:
+    vmImage: 'windows-2019'
```

Comment:
Why not ubuntu 20?

Reply:
I have used same image name as in other generate pipeline.